### PR TITLE
DDF-2886 added security services to templated docs

### PR DESCRIPTION
**Pull request by ricklarsen**
#### What does this PR do?

- Updates templated docs to include security services in integrator docs.

#### Who is reviewing it? 

@mcalcote @vinamartin @Lambeaux @jrnorth @ahoffer 

#### Choose 2 committers to review/merge the PR. 

@clockard
@coyotesqrl
@lessarderic
@pklinef
@ricklarsen - Documentation
@shaundmorris
@stustison

#### How should this be tested? (List steps with links to updated documentation)

- Build successfully and includes all expected sections in the correct order.
- "New" `-contents.adoc` files are existing content with added headers. *Content review OPTIONAL*

#### What are the relevant tickets?
[DDF-2886](https://codice.atlassian.net/browse/DDF-2886)
#### Checklist:
- [x] Documentation Updated
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.

### CODE REVIEW

File: distribution/docs/src/main/jdocs/content/_security-services/expansion-service-contents.adoc [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+Location : CA : CA USA
+Title : VP-Sales : VP-Sales VP Sales
+Title : VP-Engineering : VP-Engineering VP Engineering
+----
+
+====== security:expand
+
+The security:expand command runs the expansion service on the provided data.
+It takes an attribute and an original value, expands the original value using the current expansion service and rule set and dumps the results.
+For the rule set shown above, the expand command produces the following results:
```
Comment (shaundmorris) [inline distribution/docs/src/main/jdocs/content/_security-services/expansion-service-contents.adoc]: ❓ should this be `ruleset`, I noticed both ways of spelling it in our docs.

File: distribution/docs/src/main/jdocs/content/_security-services/external-ws-s-sts-support-contents.adoc [hunk 2]
```diff
@@ -0,0 +1,16 @@
+:type: securitySts
+:status: published
+:title: External/WS-S STS Support
+:link: _external_ws_s_sts_support
+:order: 02
```
Comment (shaundmorris) [inline distribution/docs/src/main/jdocs/content/_security-services/external-ws-s-sts-support-contents.adoc]: ❓ should there be summaries on this files?

File: distribution/docs/src/main/jdocs/content/_security-services/security-idp-intro-contents.adoc [hunk 3]
```diff
@@ -0,0 +1,32 @@
+:type: securityIdp
+:status: published
+:title: Security IdP
+:link: _encryption_service
+:order: 01
+
+The Security IdP application provides service provider handling that satisfies the http://docs.oasis-open.org/security/saml/v2.0/saml-profiles-2.0-os.pdf[SAML 2.0 Web SSO profile] in order to support external IdPs (Identity Providers) or SPs (Service Providers).
+This capability allows use of ${ddf-branding} as the SSO solution for an entire enterprise.
```
Comment (shaundmorris) [inline distribution/docs/src/main/jdocs/content/_security-services/security-idp-intro-contents.adoc]: ❓ should this be `${branding}`

File: distribution/docs/src/main/jdocs/content/_security-services/encryption-service-contents.adoc [hunk 4]
```diff
@@ -0,0 +1,28 @@
+:type: securityService
+:status: published
+:title: Encryption Service
+:link: _encryption_service
+:order: 01
+:summary: Encrypts plain text
```
Comment (ricklarsen) [inline distribution/docs/src/main/jdocs/content/_security-services/encryption-service-contents.adoc]: Yes.

File: distribution/docs/src/main/jdocs/content/_security-services/encryption-service-contents.adoc [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+
+${branding} includes an encryption service to encrypt plain text such as passwords.
+
+==== Encryption Command
+
+An encrypt security command is provided with ${branding} to encrypt text.
+This is useful when displaying password fields in a GUI.
```
Comment (ricklarsen) [inline distribution/docs/src/main/jdocs/content/_security-services/encryption-service-contents.adoc]: I changed it. I definitely don't like GUI

File: distribution/docs/src/main/jdocs/content/_security-services/expansion-service-contents.adoc (hunk 1 above)
Comment (ricklarsen) [inline distribution/docs/src/main/jdocs/content/_security-services/expansion-service-contents.adoc]: yes. I'll try to find the others and standardize.

File: distribution/docs/src/main/jdocs/content/_security-services/external-ws-s-sts-support-contents.adoc (hunk 2 above)
Comment (ricklarsen) [inline distribution/docs/src/main/jdocs/content/_security-services/external-ws-s-sts-support-contents.adoc]: Probably. They aren't used for anything by this template, but I think they are useful for future navigating.

File: distribution/docs/src/main/jdocs/content/_security-services/security-idp-intro-contents.adoc (hunk 3 above)
Comment (ricklarsen) [inline distribution/docs/src/main/jdocs/content/_security-services/security-idp-intro-contents.adoc]: probably?

File: distribution/docs/src/main/jdocs/content/_security-services/external-ws-s-sts-support-contents.adoc [hunk 6]
```diff
@@ -0,0 +1,16 @@
...
+
+===== Security STS WSS
+
+This configuration works just like the STS Client Config for the internal STS, but produces standard requests instead of the custom ${branding} ones.
+It supports two new auth types for the context policy manager, WSSBASIC and WSSPKI.
```
Comment (stustison) [inline distribution/docs/src/main/jdocs/content/_security-services/external-ws-s-sts-support-contents.adoc]: We might want to just add in a comment saying that one would use these auth types when connecting to a non-DDF STS. They can also be used with the DDF STS if you want to ignore realms.

Comment (ricklarsen) [inline distribution/docs/src/main/jdocs/content/_security-services/external-ws-s-sts-support-contents.adoc]: Added.

**Comment by ricklarsen** [comment]
build now

**Pull request merged**